csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -302,6 +304,8 @@ _Information technology — Security techniques — Vulnerability disclosure_, I
 https://www.iso.org/standard/45170.html.
 ###### [RFC3552]
 Rescorla, E. and B. Korver, "Guidelines for Writing RFC Text on Security Considerations", BCP 72, RFC 3552, DOI 10.17487/RFC3552, July 2003, https://www.rfc-editor.org/info/rfc3552.
+###### [RFC7464]
+N. Williams., "JavaScript Object Notation (JSON) Text Sequences", RFC 8259, DOI 10.17487/RFC7464, February 2015, http://www.rfc-editor.org/info/rfc7464.
```

Comment:
Based on this added reference we can add best practice advice how to stream or batch CSAF documents in high volume or near real time websocket/long polling/push scenarios (i.e. across long living connections)